.gitignore

```diff
@@ -21,3 +21,5 @@
 go.work
 
 shunit2
+
+.codegpt
```

Comment:
what's this? also, a EOF character is often preferred

Reply:
Annoyingly VSCode has started adding this to every `.gitignore` file, so I'm checking it in otherwise it will continue to appear in every diff